ui/packages/catalog-ui-search/src/main/webapp/react-component/sources/container.tsx

```diff
@@ -55,6 +55,7 @@ class SourcesSummaryContainer extends React.Component<Props, State> {
     return (
       <Sources
         sources={this.state.sources}
+        refreshSources={() => sources.fetch()}
```

Comment:
Will sources always be defined? If it is, will they always have a fetch function?

Reply:
Looks like it's the imported backbone collection instance. Should be good to go. 👍